todo_app/templates/index.html

```diff
@@ -9,10 +9,34 @@ <h1 class="display-4">To-Do App</h1>
 
   <div class="row">
     <div class="col-md-12">
+      <form method="post" action="/items">
+        <input name="new_item" type="text" />
+        <button type="submit">
+            Add item
+        </button>
+      </form>
       <h2>Items</h2>
-      <ul class="list-group mb-4">
-        <!-- List the todo items here -->
+      <ul class="list-unstyled mb-4">
+        {% for item in items %}
+          <li>
+            <form method="post" action="{{ '/items/{id}/toggle'.format(id = item.id) }}">
```

Comment:
Happy if you disagree, but I'd suggest the below is a little cleaner!

```suggestion
            <form method="post" action="/items/{{item.id}}/toggle">
```